labrad/test/test_signal.py

```diff
@@ -0,0 +1,66 @@
+import Queue
```

Comment:
It looks like this is only in tests, but perhaps it should be added to requirements.txt.  I don't have an opinion here, just wanted to point this out.